curvefs/src/mds/schedule/topoAdapter.h

```diff
@@ -115,21 +113,22 @@ struct CopySetInfo {
 
 struct MetaServerInfo {
  public:
-    MetaServerInfo()
-        : diskCapacity(0), diskUsed(0), startUpTime(0) {}
+    MetaServerInfo() : startUpTime(0) {}
     MetaServerInfo(const PeerInfo &info, OnlineState state,
-                   uint64_t capacity, uint64_t used);
+                   const MetaServerSpace &space);
 
     bool IsOnline() const;
     bool IsOffline() const;
     bool IsUnstable() const;
     bool IsHealthy() const;
+    bool IsResourceOverload() const;
+    double GetResourceUseRatioPercent() const;
+    bool IsMetaserverResourceAvaible() const;
 
-    uint64_t diskCapacity;
-    uint64_t diskUsed;
     uint64_t startUpTime;
     PeerInfo info;
     OnlineState state;
+    mutable MetaServerSpace space;
```

Comment:
if not mutable, it will compile fail at 

bool MetaServerInfo::IsResourceOverload() const {
    return space.IsResourceOverload();
}

double MetaServerInfo::GetResourceUseRatioPercent() const {
    return space.GetResourceUseRatioPercent();
}